src/components/LatestGuests/guestcontainer.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import GuestContainer from "./GuestContainer";
+import { latestAddedGuests } from "./data";
+
+it("renders correctly", () => {
```

Comment:
can we change the test name to descripe which component is getting used maybe like render Guest Container correctly

Reply:
done